amplify/backend/function/risestandstrongcb34053a/src/utils/aws-utils.js

```diff
@@ -51,7 +51,31 @@ async function postShift(shiftBody) {
     }
 }
 
+/**
+ * GET a shift from the shifts table in DynamoDB.
+ * Returns a shift. Throws error from DynamoDB if one occurs.
+ * 
+ * @param {*} shiftBody 
+ */
```

Comment:
Redo this comment to be more descriptive of what the function actually does. Mention how the shiftId is used to get a specific shift. Also change the @param from shiftBody to shiftId to be more accurate of what is actually being passed into the function.